.paket/Paket.Restore.targets

```diff
@@ -5,6 +5,11 @@
     <!-- make MSBuild track this file for incremental builds. -->
```

Comment:
Perhaps we could remove this file altogether since the projects are not dependant on paket to load

Reply:
Deleted it, and added to gitignore